CHANGELOG.rst

```diff
@@ -26,6 +26,7 @@ Changed
 - The ``Exception`` subclasses have been moved to an ``.exceptions`` module
   within each package (for example, ``qiskit.exceptions.QiskitError``). (#1600).
 
+- Removed wrappers 
```

Comment:
Can you take the chance to start the "removed" section for the release? ie

```

Removed
-------

- Removed wrappers
```

Reply:
sure